### ISSUE TITLE
Improvement: Put resource report into a table format

### ISSUE THREAD

**Issue by fosterlynn** [lightningrodlabs/rea-playspace#181]
Below is the resource report.  It is pretty hard to parse visually, would be much easier in a table format.
![Screenshot from 2022-10-30 06-59-44](https://user-images.githubusercontent.com/3776081/198877286-3ad9f39d-ad29-47ce-9b1a-373837a1b8c7.png)

**Comment by adaburrows** [comment]
Agreed. This was actually an unwritten ticket. However, I'm sorry to inform you that this needs to be broken down into two tickets, one for each location it occurs. I can do that later, if you wish.
